src/main/java/io/cdap/delta/datastream/DatastreamEventConsumer.java

```diff
@@ -248,6 +249,17 @@ private void fetchNext() throws Exception {
     }
   }
 
+  private DMLOperation.Type getOperationType(String changeType) {
```

Comment:
Can we just add some comments here as to why we check on these event types as well.

Reply:
done :)